packages/leafygreen-provider/src/index.ts

```diff
@@ -18,6 +20,6 @@ export {
   type PortalContextInterface,
   PortalContextProvider,
   usePopoverPortalContainer,
-} from './PortalContext';
-export { useBaseFontSize } from './TypographyContext';
+} from './PortalContext/PortalContext';
+export { useBaseFontSize } from './TypographyContext/TypographyContext';
```

Comment:
nit: should we export `TypographyContext/index.tsx`?

Reply:
Fair feedback, updated.